.github/workflows/sync_s3_cache.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow uploads third-party dependencies to OSSCI S3.
+# For more information and documentation, see:
+# * https://github.com/pytorch/builder/main/thirdparty_deps/README.md
+# * https://github.com/pytorch/builder/pull/1096
+# * .github/pytorch-cached-s3-dependencies.yml
+
+name: Update S3 dependencies on public OSSCI S3 buckets
+on:
+  schedule:
+    # Sync every hour
+    - cron: "*/60 * * * *"
```

Comment:
I guess just `0 * * * *` to run at the start of every hour is good enough, or just throw a random minute there :)

Reply:
Sounds reasonable, makes it more predictable, thanks!